tests/config/pytest_modules.yml

```diff
@@ -2503,6 +2503,14 @@ pairtools/stats:
   - modules/nf-core/pairtools/stats/**
   - tests/modules/nf-core/pairtools/stats/**
 
+panacus/histgrowth:
+  - modules/nf-core/panacus/histgrowth/**
+  - tests/modules/nf-core/panacus/histgrowth/**
+
+panacus/visualize:
+  - modules/nf-core/panacus/visualize/**
+  - tests/modules/nf-core/panacus/visualize/**
+
```

Comment:
```suggestion
```
This was just needed for pytest, your tests are with nf-tests and will be picked up based on their tags.